package.json

```diff
@@ -23,33 +23,38 @@
   "devDependencies": {
     "broccoli-asset-rev": "^2.2.0",
     "ember-api-store": "^1.2.13",
+    "ember-ajax": "0.7.1",
+    "ember-data": "^2.3.0",
```

Comment:
Are `ember-ajax` or `ember-data` actually needed?  I know tests used to require ember-data...